music21/instrument.py

```diff
@@ -128,11 +129,33 @@ class Instrument(base.Music21Object):
     * instrumentAbbreviation
     * midiProgram
     * midiChannel
-    * lowestNote (a note object or a string)
-    * highestNote (a note object or a string)
+    * lowestNote (a note object or a string for _written_ pitch)
+    * highestNote (a note object or a string for _written_ pitch)
+    * hardLimitOnLowest (boolean)
+    * hardLimitOnHighest (boolean)
+    * meanPitch (a note object or a string for _sounding_ pitch)
     * transposition (an interval object)
     * inGMPercMap (bool -- if it uses the GM percussion map)
     * soundfontFn (filepath to a sound font, optional)
+
+    Finally, a few clarifications on matters of range and transposition.
+
+    First, on transposing instruments, the lowestNote is for written pitch
+    (e.g. B3 for the 'English Horn')
+
+    Second, the 'hardLimit' variable is a boolean range constraint indicating that the instrument
+    cannot go beyond this pitch without non-standard modifications such as scordatura.
+    Most instruments have a hard limit on the low end
+    (e.g. a keyboard instrument's lowest key,
+    or the standard tuning of a lowest string on a string instrument)
+    but no such limit at the high side.
+    Exceptions include voices (which do not have a clearly defined lower limits) and
+    keyboard instruments (which do have clearly defined upper limits).
```

Comment:
Perhaps something to clarify here that this is still with respect to standard instrument construction? Low B foot on the flute doesn't jeopardize `hardLimitOnLowest` on your construction, nor does a Bösendorfer grand, etc...

Reply:
whoops, outdated already, nvm

Reply:
As above